esmvalcore/cmor/check.py

```diff
@@ -985,16 +985,16 @@ def cmor_check_metadata(cube,
         CMOR definitions to use.
     mip:
         Variable's mip.
-    short_name: basestring
-        Variable's short name.
+    cmor_name: basestring
```

Comment:
```suggestion
    cmor_name: str
```
`basestring` can be replaced with `str` everywhere, I believe `basestring` is a remnant from when we still supported Python 2.